### DISCUSSION TITLE
Default tab is not updated

### DISCUSSION THREAD

**Discussion by msolomos** [kinghat/tabbed-card#142] [Q&A] [answered]
hello,
I've tried your tabbed-card. However, it's not updated at all

here is the code I am using

`type: custom:tabbed-card-programmable
options:
  defaultTabIndex: >
    "{% if states('group.skylight_windows_status') == 'on' %}2{% else %}0{%
    endif %}"
styles:
  '--mdc-tab-text-label-color-default''''''': grey
  '--mdc-tab-color-default': grey
  '--mdc-typography-button-font-size': 11px
tabs:
  - attributes:
      label: ΘΕΡΜ.
      icon: mdi:thermometer
    card:
      type: glance
      entities:
        - entity: sensor.temperature_baby_room_calibrated
          name: Kids
        - entity: sensor.temperature_master_calibrated
          name: Master
        - entity: sensor.hue_motion_sensor_1_temperature
          name: Κουζίνα
        - entity: sensor.temperature_hallway_calibrated
          name: Hallway
        - entity: sensor.temperature_and_humidity_sensor_84ab_temperature
          name: WC
  - attributes:
      label: ΥΓΡΑΣΙΑ
      icon: mdi:water-percent
    card:
      type: glance
      entities:
        - entity: sensor.humidity_baby_room_calibrated
          name: Kids
        - entity: sensor.humidity_master_calibrated
          name: Master
        - entity: sensor.humidity_attic_calibrated
          name: Σοφίτα
        - entity: sensor.temperature_and_humidity_sensor_84ab_humidity
          name: WC
  - attributes:
      label: ΦΕΓΓΙΤΕΣ
      icon: mdi:water-percent
    card:
      type: glance
      entities:
        - entity: cover.attic_skylight
          name: Skylight 1
        - entity: cover.attic_skylight_2
          name: Skylight 2
`

However, when I turn on the skylights and the active tab should be no. 2, it doesn't work. It shows 0.

would you please have a look?

**Comment by ildar170975** [comment]
You posted an unformatted code which is impossible to analyse, can only guess.

**Reply by ildar170975** [reply, marked as answer]
And you posted an issue in a wrong repo. "type: custom:tabbed-card-programmable" - a fork.

**Comment by msolomos** [comment]
please accept my apologies. For posting in a wrong repo and for an unformatted code.

**Reply by ildar170975** [reply]
You are welcome! Next time please post a formatted code.
